includes/fields.php

```diff
@@ -155,6 +154,15 @@ function share_light_field_widget_form(&$form, &$form_state, $field, $instance,
   return $element;
 }
 
+/**
+ * Updates the form state of the share_light block by transforming its options.
+ *
+ * Original structure `$values['channels']`:
+ * {"toggle_CHANNEL1": 1, "options_CHANNEL1": {"OPTION1": "VALUE1",...},...}`
+ *
+ * Transformed structure `$values['channels']`:
+ * {"CHANNEL1": {"toggle": True, "OPTION1": "VALUE1",...},...}`
+ */
 function _share_light_transform_options($element, &$form_state, $form) {
```

Comment:
This is actually an `#element_validate` callback for the field widget. There is no such thing as "the form state of the share_light block".